taichi/backends/vulkan/vulkan_device.h

```diff
@@ -635,9 +636,9 @@ class VulkanDevice : public GraphicsDevice {
   VkQueue graphics_queue_;
   uint32_t graphics_queue_family_index_;
 
-  unordered_map<std::thread::id, std::unique_ptr<VulkanStream>> compute_stream_;
-  unordered_map<std::thread::id, std::unique_ptr<VulkanStream>>
-      graphics_stream_;
+  struct ThreadLocalStreams;
```

Comment:
Note that I have to hide `VulkanStream`, because it is not marked as `TI_DLL_EXPORT`. This is required on Windows.